config/data/holes.toml

```diff
@@ -3969,6 +3975,7 @@ preamble = '''
 <p>Print the first <b>1,000</b> decimal digits of 2π=τ (tau).
 {{- end -}}
 '''
+aliases = ['pi']
```

Comment:
I really like this stuff, because I like to type these to navigate, but I don't bother with ones that aren't easy to type.

Reply:
> I really like this stuff, because I like to type these to navigate, but I don't bother with ones that aren't easy to type.

Yeah, same.